services/company.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Company } from "../models";
+import { ICompany } from "../models/interface/company.model.interface";
+
+export class CompanyService {
+
+    public async createCompany(company: ICompany) {
+        try {
+
+            const companyResponse = await new Company(company);
+            return companyResponse.save();
+
+        } catch(error) {
+            console.error(`CompanyService createCompany error: ${error}`);
+            throw new Error();
+        }
+    }
+
+    public async updateCompany(company: any) {

```

Comment:
why `any` here?

Reply:
company update functionality will be implemented in a separate task